proto/options.proto

```diff
@@ -310,6 +310,9 @@ message Costing {
     uint32 fixed_speed = 80;
     uint32 axle_count = 81;
     float use_lit = 82;
+    oneof has_disable_hierarchy_pruning {
+      bool disable_hierarchy_pruning = 83;
+    }
```

Comment:
doesn't need to be a oneof, since the default will be false

Reply:
done!